src/pages/Compare/index.spec.tsx

```diff
@@ -546,45 +459,34 @@ describe(__filename, () => {
     });
   });
 
-  // This could happen when a keyboard navigation updates the selected path.
-  it('does not dispatch viewVersionFile() on update if the query string contains a `path` that is different than `version.selectedPath`', () => {
-    const { dispatchSpy, root } = loadDiffAndRender();
+  it('does not dispatch anything on mount when an API error has occured', () => {
```

Comment:
It's because we don't do anything else actually. `compareInfo` is `null`, which is not `undefiend` either, so we execute the whole `loadData` method but there is no path we can trigger.

Reply:
I see. Huh, that's interesting. I guess you don't need the `null` checking if statement at all, heh, but we should keep it to make the intent explicit.